jinja2/includes/icons/emojis/thumbs-up.svg

```diff
@@ -1 +1 @@
-<svg class="icon icon-emoji-thumbs-up" width="25" height="25" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><text transform="translate(-15 -13)" fill="#262D2D" fill-rule="evenodd" font-family="AppleColorEmoji, Apple Color Emoji" font-size="25"><tspan x="15" y="36">👍</tspan></text></svg>
+<svg viewBox="0 0 914.286 1024" class="icon icon-thumbs-up" aria-hidden="true"><path class="path0" stroke="inherit" d="M146.286 768c0-20-16.571-36.571-36.571-36.571-20.571 0-36.571 16.571-36.571 36.571 0 20.571 16 36.571 36.571 36.571 20 0 36.571-16 36.571-36.571zm91.428-292.571v365.714c0 20-16.571 36.571-36.571 36.571H36.572c-20 0-36.571-16.571-36.571-36.571V475.429c0-20 16.571-36.571 36.571-36.571h164.571c20 0 36.571 16.571 36.571 36.571zm676.572 0c0 30.286-12 62.857-31.429 85.143 6.286 18.286 8.571 35.429 8.571 43.429 1.143 28.571-7.429 55.429-24.571 78.286 6.286 21.143 6.286 44 0 66.857-5.714 21.143-16.571 40-30.857 53.714 3.429 42.857-6.286 77.714-28 103.429-24.571 29.143-62.286 44-112.571 44.571h-73.714c-81.714 0-158.857-26.857-220.571-48-36-12.571-70.286-24.571-90.286-25.143-19.429-.571-36.571-16.571-36.571-36.571V474.858c0-18.857 16-34.857 34.857-36.571 21.143-1.714 76-69.714 101.143-102.857 20.571-26.286 40-50.857 57.714-68.571 22.286-22.286 28.571-56.571 35.429-89.714 6.286-33.714 13.143-69.143 37.714-93.143 6.857-6.857 16-10.857 25.714-10.857 128 0 128 102.286 128 146.286 0 46.857-16.571 80-32 109.714-6.286 12.571-12 18.286-16.571 36.571h158.286c59.429 0 109.714 50.286 109.714 109.714z"/></svg>
```

Comment:
GitHub says this is an invalid SVG. And maybe ``aria-hidden`` should be removed, or added to ``thumbs-down``.